target/linux/layerscape/armv8_64b/config-5.4

```diff
@@ -219,8 +219,6 @@ CONFIG_CRYPTO_RNG2=y
 CONFIG_CRYPTO_ZSTD=y
 CONFIG_DCACHE_WORD_ACCESS=y
 CONFIG_DEBUG_BUGVERBOSE=y
-CONFIG_DEBUG_INFO=y
-# CONFIG_DEBUG_INFO_REDUCED is not set
```

Comment:
The CONFIG_DEBUG_INFO_REDUCED change should be moved to the subsequent patch, as this one should actually not be cosmetic here.

Reply:
Yes, it's cleaner located in the next commit.